Open Judge System/Workers/OJS.Workers.ExecutionStrategies/NodeJsExecuteAndRunAsyncJsDomTestsWithReactExecutionStrategy.cs

```diff
@@ -41,27 +41,27 @@ public class NodeJsExecuteAndRunAsyncJsDomTestsWithReactExecutionStrategy :
         {
             if (!File.Exists(sinonJsDomModulePath))
             {
-                throw new ArgumentException($"SinonPackaged not found in: {sinonJsDomModulePath}", nameof(sinonJsDomModulePath));
+                throw new ArgumentException($@"SinonPackaged not found in: {sinonJsDomModulePath}", nameof(sinonJsDomModulePath));
```

Comment:
Changes the text to verbatim string, Resharper believes verbatim strings should be used to avoid escaping symbols and so forth by mistake.